chapter_multilayer-perceptrons/kaggle-house-price.md

```diff
@@ -674,7 +674,7 @@ will simplify uploading the results to Kaggle.
 #@tab all
 def train_and_pred(train_features, test_feature, train_labels, test_data,
                    num_epochs, lr, weight_decay, batch_size):
-    net = get_net()
+    net = get_net(weight_decay)
```

Comment:
`tab all` means it shares with mxnet and pytorch. if you only change the tf version, it will lead error for other frameworks.  I suggest don't change function apis

Reply:
Got it. I didn't realize that this is in a `tab all` block. I have reverted the API change.